etcd-backup-operator/hack/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
```

Comment:
This pr should be  purely removing the extra function. Don't couple and mix  extra stuff.

Reply:
kk, I'll have another pr that does the build stuff.